test/legacy/roles/test_digital_ocean/tasks/sshkey.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Create ssh key
+  digital_ocean_sshkey:
+    name: test-key1
+    ssh_pub_key: "{{ dummy_ssh_pub_key }}"
+    oauth_token: "{{ digitalocean_oauth_token }}"
+  register: result
+
+- name: Verify that SSH key was created
+  assert:
+    that:
+      - "result.changed"
+
+- name: "Delete ssh key"
+  digital_ocean_sshkey:
+    state: "absent"
+    fingerprint: "{{ result.data.ssh_key.fingerprint }}"
+    oauth_token: "{{ digitalocean_oauth_token }}"
```

Comment:
Doesn't this need a `register: result`?

Reply:
True, also missing in **floating_ip.yml**. I pushed a fixed but I was wondering if I should re-use *result* or use another variable...